Sources/ProjectDescription/EnvironmentVariable.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// It represents an environment variable that is passed when running a scheme's action
+public struct EnvironmentVariable: Equatable, Codable, Hashable, ExpressibleByStringLiteral {
+    // MARK: - Attributes
+
+    /// The value of the environment variable
+    public let value: String
+    /// Whether the variable is enabled or not
+    public let isEnabled: Bool
+
+    // MARK: - Init
+
+    public init(value: String, isEnabled: Bool) {
+        self.value = value
+        self.isEnabled = isEnabled
+    }
+
+    public init(stringLiteral value: String) {
```

Comment:
I'd add documentation here to indicate that it initialises it being enabled by default.

Reply:
That’s the same behavior as it has been before. But I’m happy to document this. Please let me know where and how I can add the documentation.